src/System.Management.Automation/help/HelpUtils.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Management.Automation;
+
+namespace System.Management.Automation
+{
+    internal class HelpUtils
+    {
+        /// <summary>
+        /// Get the path to $HOME
+        /// </summary>
+        internal static string GetUserHomeHelpSearchPath()
```

Comment:
This could be made a static property with a static field that caches the path.

Reply:
Fixed.